deployment/fedora-package-x64/pkg-src/jellyfin.spec

```diff
@@ -12,28 +12,36 @@ Release:        1%{?dist}
 Summary:        The Free Software Media Browser
 License:        GPLv2
 URL:            https://jellyfin.media
-Source0:        %{name}-%{version}.tar.gz
-Source1:        jellyfin.service
-Source2:        jellyfin.env
-Source3:        jellyfin.sudoers
-Source4:        restart.sh
-Source5:        jellyfin.override.conf
-Source6:        jellyfin-firewalld.xml
+# Jellyfin Server tarball created by `make -f .copr/Makefile srpm`, real URL ends with `v%{version}.tar.gz`
+Source0:        https://github.com/%{name}/%{name}/archive/%{name}-%{version}.tar.gz
```

Comment:
Is it missing literal `v` in url, before `%{version} `? Comment says it should be there

Reply:
These aren't actually used since the source tarball is made by `.copr/Makefile`, though you are right about them being wrong.